src/app/app.module.ts

```diff
@@ -29,7 +31,9 @@ import { MatCardModule } from '@angular/material/card';
     FormsModule,
     BrowserAnimationsModule,
     MatToolbarModule,
-    MatCardModule
+    MatCardModule,
+    RouterModule.forRoot([]),
```

Comment:
We already export all routes from app-routing module as our root. Please use that instead of declaring directly in app-module file.